packages/rest/tsup.config.ts

```diff
@@ -2,6 +2,12 @@ import { esbuildPluginVersionInjector } from 'esbuild-plugin-version-injector';
 import { createTsupConfig } from '../../tsup.config.js';
 
 export default createTsupConfig({
-	entry: ['src/index.ts', 'src/strategies/*.ts'],
+	entry: [
+		'src/agnostic.ts',
+		'src/web.ts',
+		'src/strategies/undiciRequest.ts',
```

Comment:
no need to call out each strategy individually (alos fetchRequest.ts doesn't exist lol)